packages/core/src/ivy-script-client.ts

```diff
@@ -1,14 +1,14 @@
 import { createWebSocketConnection, type Connection } from '@axonivy/jsonrpc';
 
 export namespace IvyScriptLanguage {
-  export async function startWebSocketClient(url: string, isMonacoReady: Promise<boolean>): Promise<any> {
+  export async function startWebSocketClient(url: string, isMonacoReady: Promise<any>): Promise<any> {
```

Comment:
why not Promise<void>?

Reply:
> why not Promise?

I'm not sure I fully understand but the return type that we typically use here is of `monacoEditorReact.Monaco` but since we are not actually awaiting the result in this function, I typed it any (instead of `void` or `boolean` previously) mainly to ease changes and avoid conversions like `MonacoEditorUtil.getInstance().then(() => {})` but also highlight the fact that we do not care about the return type.

Reply:
sorry not everything was posted... 
I wanted to write `Why not Promise<void>`?

Reply:
Does my post above answer the question? Basically we want to avoid unnecessary conversions.